### PR TITLE
[WIP] add `expect` method to fg builtin 

### PR DESCRIPTION
**Pull request by SukitOwl**
Can set expectation for do request with fostlib::ua

### CODE REVIEW

File: test/test-views/tests.hpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#include <fost/urlhandler>
+
+
+namespace test {
+
+
+    namespace view {
+
+
+        extern const fostlib::urlhandler::view &chain_request;
```
Comment (KayEss) [inline test/test-views/tests.hpp:21]: I don't think we need this file. We've deprecated all of these variables and removed them all elsewhere

File: test/test-views/chain-request.cpp [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    Distributed under the Boost Software License, Version 1.0.
+    See <http://www.boost.org/LICENSE_1_0.txt>
+ */
+
+
+#include <fost/http>
+#include <fost/ua/exceptions.hpp>
+#include "tests.hpp"
+#include <iostream>
```
Comment (KayEss) [inline test/test-views/chain-request.cpp:12]: This shouldn't be here should it?

File: test/fg-expect/view.chain-request.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "webserver" : {
+        "views/chain.request": {
+            "view": "test.chain.request"
```
Comment (KayEss) [inline test/fg-expect/view.chain-request.json:4]: You can embed this in the .fg script as well. May be simpler

Comment (KayEss) [reply test/fg-expect/view.chain-request.json:4]: But actually, why not just use `test.chain.request` directly in the .fg test script?

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,6 @@
+2020-06-30 Sukit Saeo <sukit_owl@hotmail.com>
```
Comment (KayEss) [inline ChangeLog:1]: There's two spaces between the fields

File: test/fg-expect/view.chain-request.json (hunk 3 above)
Comment (SukitOwl) [reply test/fg-expect/view.chain-request.json]: Just forget we can do that

**Pull request merged**